workflows/exomeseq.cwl

```diff
@@ -14,26 +14,57 @@ inputs:
   # Read pairs, fastq format
   read_pairs:
       type: { type: array, items: { type: array, items: File } }
-  reference_reads: File[]
+  reference_reads:
+    type: File[]
+    secondaryFiles:
+    - ^.bai
   # reference genome, fasta
-  reference_genome: File
+  reference_genome:
+    type: File
+    secondaryFiles:
+    - .amb
+    - .ann
+    - .bwt
+    - .pac
+    - .sa
+    - .fai
+    - ^.dict
   # Number of threads to use
   threads: int?
   # Read Group annotation
   read_group_header: string
   # GATK
-  GATKJar: File
-  knownSites: File[] # vcf files of known sites, with indexing
+  GATKJar:
+    type: File
```

Comment:
No, toil was fine with the condensed version. 
I just made the File inputs lines consistent. 
I don't know if there is a clean way to put the secondaryFiles on one line.